scipy/stats/__init__.py

```diff
@@ -117,6 +117,7 @@
    triang            -- Triangular
    truncexpon        -- Truncated Exponential
    truncnorm         -- Truncated Normal
+   truncweibull_min  -- Truncated minimum Weibull distribution
```

Comment:
There are a few other things that need to be done on the [checklist](http://scipy.github.io/devdocs/dev/contributor/adding_new.html#adding-a-new-statistics-distribution).

One doctest error would be fixed by updating the number of continuous distributions in the tutorial document.
The other would be fixed by adding parameters to `_distr_params.py`

Reply:
The above issues should be fixed now.

Reply:
Yes, looks like it.